sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClientOptions.cs

```diff
@@ -49,14 +50,22 @@ public class EventProcessorClientOptions
 
         /// <summary>
         ///   The maximum amount of time to wait for an event to become available for a given partition before emitting
-        ///   a <c>null</c> event.
+        ///   an empty event.
         /// </summary>
         ///
         /// <value>
         ///   If <c>null</c>, the processor will wait indefinitely for an event to become available; otherwise, a value will
         ///   always be emitted within this interval, whether an event was available or not.
         /// </value>
         ///
+        /// <remarks>
+        ///   When set, if no events are received before the timeout, <see cref="EventProcessorClient.ProcessEventAsync" />
+        ///   is called with an <see cref="ProcessEventArgs" /> without any event data. <see cref="ProcessEventArgs.HasEvent" />
```

Comment:
nit:  This reads a bit awkward to me.   Maybe consider something like:  `... is called with a <see cref="ProcessEventArgs" /> instance that does not contain any event data.  The <see cref="ProcessingEventArgs.HasEvent" /> property is intended to help detect this.`

Reply:
I like this wording much better.  Thanks!